cirq-google/cirq_google/calibration/phased_fsim_test.py

```diff
@@ -874,3 +875,40 @@ def test_options_phase_corrected_override():
         ).zeta_chi_gamma_correction_override()
         == PhasedFSimCharacterization()
     )
+
+
+def test_try_convert_gate_to_fsim():
```

Comment:
Could you expand the `test_phase_calibrated_fsim_gate_compensated` and `test_try_convert_sqrt_iswap_to_fsim_converts_correctly` with the newly supported gates, so that we have a checks that test the resulting untaries as well?

(Tests based on `assert_allclose_up_to_global_phase` or `np.allclose(cirq.unitary(fsim), expected_unitary)`)